docs/sandboxes.rst

```diff
@@ -0,0 +1,25 @@
+Sandboxes
```

Comment:
We should also remove sandboxes from `postprocess.rst` and probably add a mention to the main readme.

Reply:
Updated in 1382c15